src/T2DMap.cpp

```diff
@@ -2594,20 +2594,19 @@ void T2DMap::mousePressEvent(QMouseEvent* event)
 
             TRoom* room = mpMap->mpRoomDB->getRoom(mCustomLinesRoomFrom);
             if (room) {
-                QAction* action = new QAction("undo", this);
+                QAction* action = new QAction(tr("Undo", "Menu option to undo the last point in the mapper"), this);
```

Comment:
Actually should be:
`"Menu option to undo the last point (line segment) added to a new custom exit line"`